test/test_rcm2points.py

```diff
@@ -68,131 +74,97 @@
 
 # run tests
 class Test_rcm2points(ut.TestCase):
-    """
-    Test_rcm2points
-    This unit test covers the nominal, nan, and msg cases of 64 and 32 bit float input for rcm2points
-    """
+    """Test_rcm2points This unit test covers the nominal, nan, and msg cases of
+    64 and 32 bit float input for rcm2points."""
 
     def test_rcm2points_float64_nom_opt0(self):
         nt.assert_array_almost_equal(
             fo_nom_opt0_expected,
-            geocat.f2py.rcm2points(lat2d,
-                                   lon2d,
-                                   fi_nom.astype(np.float64),
-                                   lat,
-                                   lon,
-                                   opt=0))
+            rcm2points(lat2d, lon2d, fi_nom.astype(np.float64), lat, lon,
+                       opt=0))
 
     def test_rcm2points_float64_nom_opt2(self):
         nt.assert_array_almost_equal(
             fo_nom_opt2_expected,
-            geocat.f2py.rcm2points(lat2d,
-                                   lon2d,
-                                   fi_nom.astype(np.float64),
-                                   lat,
-                                   lon,
-                                   opt=2))
+            rcm2points(lat2d, lon2d, fi_nom.astype(np.float64), lat, lon,
+                       opt=2))
```

Comment:
I do like this linter change. We should talk about all the differences we're seeing between the two styles that should have been the same that we are using.

Reply:
OBE

Reply:
> I do like this linter change. We should talk about all the differences we're seeing between the two styles that should have been the same that we are using.

Now, I spent more time on it, and here are my findings:

There is no difference between the behaviors of our older YAPF config or that in this PR. The main difference is that we called `geocat.f2py.rcm2points` before, but now we converted it to `rcm2points` plus an initial `import geocat.f2py`. That being said, I ran both pre-commit yapf and our older yapf (i.e. `python3 -m yapf --in-place --recursive --style google .` of the `pip3`-installed yapf), and both made the exact same changes as follows:

```
rcm2points(lat2d, lon2d, fi_nom.astype(np.float64), lat, lon,
           opt=2))
```

```
geocat.f2py.rcm2points(lat2d, 
                       lon2d, 
                       fi_nom.astype(np.float64), 
                       lat, 
                       lon,
                       opt=2))
```

It looks strange to me :)